backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserPermission.java

```diff
@@ -11,5 +11,6 @@
     EDIT_ORGANIZATION,
     START_TEST,
     UPDATE_TEST,
-    SUBMIT_TEST;
+    SUBMIT_TEST,
+    EXPORT_TEST_EVENT;
```

Comment:
👍  we should probably put some javadocs on these individual permissions some day soon.